ha/core/cluster/cluster_manager.py

```diff
@@ -433,7 +433,7 @@ def is_status_present(self, element, element_id: str = HEALTH_STATUSES.UNKNOWN.v
                     break
         else:
             for key in self._status_dict:
-                if re.search(f"{element}/.+\w/health", key):
+                if re.search(f"{element}/.+/health", key):
```

Comment:
@ajaysrivas @ArchanaLimaye I saw this issue on the setup where node ids are single digit e.g. '0', then that node status was not returned. I guess this "\w" was not required, hence removed. With this change the node status is returned fine.